### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by jlewi**
* Increase the disk size for minikube because people have encountered
  problems starting our Jupyter images.

* Explain why we use /home/jovyan for the PD

* Provide some instructions for troubleshooting problems spawning notebooks.

Related to #594

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/613)

### CODE REVIEW

File: user_guide.md [hunk 1]
```diff
@@ -411,14 +447,22 @@ tmpfs                                                           15444244       0
...
+
 ```
-Warning  FailedScheduling  8s (x22 over 5m)  default-scheduler  0/1 nodes are available: 1 Insufficient memory.
+minikube start --cpus 4 --memory 8096 --disk-size=40g
 ```
-then try recreating your Minikube cluster (and re-apply Kubeflow using Ksonnet) with more resources (as your environment allows):
+
+  * Minikube by default allocates 2048Mb of RAM for its VM which is not enough
+    for JupyterHub.
+  * The larger disk is needed to accomodate Kubeflow's Jupyter images which
```
Comment (willingc) [inline user_guide.md]: Let's reword this to make clear that the size of the images are due to the ML libraries and third party packages being installed along with Jupyter. Thanks.

File: user_guide.md [hunk 2]
```diff
@@ -440,6 +484,40 @@ kubectl create clusterrolebinding default-admin --clusterrole=cluster-admin --us
 If you're using GKE, you may want to refer to [GKE's RBAC docs](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control) to understand
 how RBAC interacts with IAM on GCP.
 
+### Problems spawning Jupyter pods
+
+If you're having trouble spawning Jupyter notebooks; check that the pod is getting
```
Comment (pdmack) [inline user_guide.md]: nit: comma, not semi-colon

File: user_guide.md [hunk 3]
```diff
@@ -440,6 +484,40 @@ kubectl create clusterrolebinding default-admin --clusterrole=cluster-admin --us
...
+scheduled
+
+```
+kubectl -n ${NAMESPACE} get pods
+```
+
+  * Look for pods whose name starts with Juypter
```
Comment (pdmack) [inline user_guide.md]: nit: starts with jupyter (lower case)

File: user_guide.md (hunk 1 above)
Comment (jlewi) [reply user_guide.md]: Done

File: user_guide.md (hunk 3 above)
Comment (jlewi) [reply user_guide.md]: Done

File: user_guide.md (hunk 2 above)
Comment (jlewi) [reply user_guide.md]: Done
